src/error/__tests__/GraphQLError-test.ts

```diff
@@ -163,7 +163,7 @@ describe('printError', () => {
     );
     const opA = docA.definitions[0];
     invariant(opA.kind === Kind.OBJECT_TYPE_DEFINITION && opA.fields);
-    const fieldA = opA.fields[0];
+    const fieldA = ((opA as ObjectTypeDefinitionNode).fields ?? [])[0];
```

Comment:
It will screw our coverage please run `npm run testonly:cover` to see coverage changes.
Please use TS `!` for that.